app/src/test/java/org/mozilla/rocket/nightmode/NightModeTestUtils.kt

```diff
@@ -0,0 +1,27 @@
+package org.mozilla.rocket.nightmode
+
+import android.app.Activity
+import android.view.WindowManager
+
+import org.mozilla.focus.utils.Settings
+import org.robolectric.Robolectric
+import org.robolectric.RuntimeEnvironment
+
+import org.junit.Assert.assertTrue
+
+internal object NightModeTestUtils {
+
+    private val TARGET_BRIGHTNESS = 30.0f
+
+    fun <T : Activity> testActivityBrightness(classname: Class<T>): T {
```

Comment:
This more like UI test. Not unit test. Please remove the tests and let's land the code first. We'll discuss the unit test later when we have time. Thank you!

Reply:
It's done.